### DISCUSSION TITLE
why file not write heic

### DISCUSSION THREAD

**Discussion by lasmcmoasmoca** [ImageMagick/ImageMagick#5056] [Help]
example: convert 95fa1caaa6a04c679e839aef84384cd3.heic test.heic 
error :
convert: no encode delegate for this image format `HEIC' @ error/constitute.c/WriteImage/1392.

my centOS version :
CentOS Linux release 7.9.2009 (Core)

ImageMagick version:
Version: ImageMagick 7.1.0-31 Q16-HDRI x86_64 90160a408:20220417 https://imagemagick.org
Copyright: (C) 1999 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI OpenMP(4.5)
Delegates (built-in): bzlib fontconfig freetype heic jbig jng jpeg lzma png tiff webp x xml zlib
Compiler: gcc (11.2)

execute "convert -list format"
HEIC  r--   High Efficiency Image Format (1.12.0)
HEIF  r--   High Efficiency Image Format (1.12.0)

**Comment by yunxuanwang** [comment]
I solved this problem by install libx265.  Seems if you don't install libx265, only has libde265, libheif won't support heic encode.
